commit acabdd34745c02fbc950ddfdbfcb06d58b4f2414 (refs/changes/63/863/3)
Author: June Yi <june.yi@samsung.com>
Date:   2016-11-11T17:29:05+00:00 (2 years, 11 months ago)
    
    joyent/triton#219 sdc-usbkey mount fails in CoaL on KVM
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Chris Burroughs <chris.burroughs@joyent.com>

diff --git a/tools/coal-linux-kvm-setup b/tools/coal-linux-kvm-setup
index 43ef238..b7ef212 100755
--- a/tools/coal-linux-kvm-setup
+++ b/tools/coal-linux-kvm-setup
@@ -85,14 +85,6 @@ if [[ ("$#" -eq 1) && ("${1}" == "--use-raw-image") ]]; then
     DATA_PLANE_ENABLED=$(${EMULATOR} -device virtio-blk-pci,? |& grep -c x-data-plane || true)
     if [[ "${DATA_PLANE_ENABLED}" != "0" ]]; then
          QEMU_COMMANDLINE="
-           <qemu:commandline>
-             <qemu:arg value='-set'/>
-             <qemu:arg value='device.virtio-disk0.scsi=off'/>
-           </qemu:commandline>
-           <qemu:commandline>
-             <qemu:arg value='-set'/>
-             <qemu:arg value='device.virtio-disk0.x-data-plane=on'/>
-            </qemu:commandline>
            <qemu:commandline>
              <qemu:arg value='-set'/>
              <qemu:arg value='device.virtio-disk1.scsi=off'/>
@@ -226,8 +218,8 @@ cat > ${VIRSH_DOMAIN_XML} << EOF
         <disk type='file' device='disk'>
             <driver name='qemu' type='${QEMU_IMAGE_FORMAT}' cache='none' io='native'/>
             <source file='/var/lib/libvirt/images/sdc-headnode-usb.${QEMU_IMAGE_FORMAT}'/>
-            <target dev='hda' bus='virtio' />
-            <address type='pci' domain='0x0000' bus='0x00' slot='0x06' function='0x0'/>
+            <target dev='hda' bus='ide' />
+            <address type='drive' controller='0' bus='1' target='0' unit='0'/>
         </disk>
         <disk type='file' device='disk'>
             <driver name='qemu' type='${QEMU_IMAGE_FORMAT}' cache='none' io='native'/>
